frontend/src/App.vue

```diff
@@ -1,14 +1,20 @@
 <script setup>
 import SideBar from './layouts/SideBar/SideBar.vue';
+import TheHeader from './layouts/Header/TheHeader.vue';
 import { useUserStore } from '@/stores/user';
 
 const userStore = useUserStore();
 </script>
 
 <template>
   <div class="app">
-    <side-bar v-if="userStore.isLoggedIn" />
-    <router-view />
+    <div v-if="userStore.isLoggedIn">
+      <side-bar />
+    </div>
```

Comment:
Nitpick: is this `div` necessary? Is the only reason it is here because of the `v-if`?

Reply:
Using v-if in this div is here only so it's not included in auth page. It is not necessary, I removed it. It was used in one of the approaches to achieve the desired layout. Without it sidebar was taking 50% of the page and later on it stayed as quick fix. Sidebar width is set to a constant now and that fixes that.